environments.yml

```diff
@@ -45,3 +45,9 @@ scrapers:
     paramstore: /bobsled/scrapers/NEW_YORK_API_KEY 
   - variable: DC_API_KEY
     paramstore: /bobsled/scrapers/DC_API_KEY
+  - variable: AWS_ACCESS_KEY_ID
+    paramstore: /bobsled/backups/AWS_ACCESS_KEY_ID
```

Comment:
I think `/bobsled/scrapers/` is the more appropriate path here.

Reply:
Roger that